src/nat/eval/llm_validator.py

```
@@ -0,0 +1,333 @@
+        return (llm_name, None, None)
+
+    except TimeoutError:
+        error_msg = f"Validation timed out after {VALIDATION_TIMEOUT_SECONDS}s"
+        logger.warning("LLM '%s' validation timed out", llm_name)
+        return (llm_name, "warning", _truncate_error_message(error_msg))
+
+    except (KeyboardInterrupt, SystemExit):
+        # Don't catch system-level interrupts
+        raise
+
+    except Exception as invoke_error:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)